search-resources-api/src/main/java/no/unit/nva/search/ElasticSearchHighLevelRestClient.java

```diff
@@ -114,15 +110,19 @@ public SearchResourcesResponse searchSingleTerm(String term, int results) throws
     }
 
     private SearchResponse doSearch(String term, int results) throws IOException {
+        final SearchRequest searchRequest = getSearchRequest(term, results);
+        SearchResponse searchResponse = elasticSearchClient.search(searchRequest, RequestOptions.DEFAULT);
+        return searchResponse;
+    }
+
+    private SearchRequest getSearchRequest(String term, int results) {
```

Comment:
maybe just inline the static import of queryStringQuery

Reply:
inlined